ovirt-guest-agent-fedora/ovirt-container-hibernate.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/usr/bin/nsenter --mount=/proc/1/ns/mnt -- /usr/share/ovirt-guest-agent/hibernate $@
```

Comment:
Doesn't this mean that the /usr/share/overr-guest-agent/hibernate is on the host not in the container?